docs/lib/graphqlapi/fragments/flutter/advanced-workflows/10_example.md

```diff
@@ -0,0 +1,28 @@
+```dart
+try{
```

Comment:
```suggestion
try {
```
We've been lax on formatting in the code but should be dogmatic about it in documentation.